src/app/find/locationDetails/CategoryCard.jsx

```diff
@@ -1,150 +1,40 @@
+// all services that a shelter provides within that category
+class CategoryCard extends Component {
+  state = {
+    collapsed: false,
   }
 
-  const dayNumberToName = weekday => DAYS[weekday - 1];
-
-  const formatHour = time => moment(time, 'HH:mm:ss').format('LT').replace(':00 ', ' ');
-  const formatHours = (opens, closes) => `${formatHour(opens)} to ${formatHour(closes)}`;
-
-  const formatRange = ({ start, end }) => {
-    if (end === start) {
-      return dayNumberToName(start);
-    }
-    if (end === start + 1) {
-      return `${dayNumberToName(start)} & ${dayNumberToName(end)}`;
-    }
-    return `${dayNumberToName(start)} to ${dayNumberToName(end)}`;
-  };
-
-  const orderedSchedule = schedule.sort(({ weekday: day1 }, { weekday: day2 }) => day1 - day2);
-
-  const daysGroupedByHours = orderedSchedule.reduce((grouped, day) => {
-    const hoursString = formatHours(day.opens_at, day.closes_at);
-    return { ...grouped, [hoursString]: [...(grouped[hoursString] || []), day.weekday] };
-  }, {});
-
-  const groupStrings = Object.keys(daysGroupedByHours).map((hoursString) => {
-    const dayRanges = [];
-    const remainingWeekdays = daysGroupedByHours[hoursString];
-
-    while (remainingWeekdays.length) {
-      const currentDayRange = dayRanges[dayRanges.length - 1];
-      const day = remainingWeekdays.shift();
-
-      if (currentDayRange && currentDayRange.end === day - 1) {
-        currentDayRange.end = day;
-      } else {
-        dayRanges.push({ start: day, end: day });
-      }
-    }
-
-    return `${dayRanges.map(formatRange).join(', ')} ${hoursString}`;
-  });
-
-  return `Open ${groupStrings.join(', ')}`;
-};
-
-const renderService = (service, isLastItem) => {
-  const coronavirusInfo = service.EventRelatedInfos &&
-    service.EventRelatedInfos.filter(({ event }) => event === OCCASIONS.COVID19);
-
-  const schedule = service.HolidaySchedules &&
-    service.HolidaySchedules.filter(({ occasion, closed }) => occasion === OCCASIONS.COVID19);
-  const isScheduleKnown = schedule && schedule.length;
-
-  let isClosed = false;
-  let openDays;
-  if (isScheduleKnown) {
-    openDays = schedule.filter(({ closed }) => !closed);
-
-    if (!openDays.length) {
-      isClosed = true;
-    }
+  handleCollapse = () => {
+    this.setState({ collapsed: !this.state.collapsed });
   }
 
-  return (
-    <div
-      key={service.id}
-      style={{
-        borderBottom: isLastItem ? '0' : '1px solid #DADADA',
-        paddingBottom: '1.3vh',
-      }}
-    >
-      <div size="medium" className="specificServiceHeaders">
-        {service.name || service.Taxonomies[0].name}
-        {isClosed && <span className="coronavirusInfo"> (suspended)</span>}
-      </div>
-
-      {!!(coronavirusInfo && coronavirusInfo.length) && (
-        <InfoItem coronavirus icon="exclamation-triangle">
-          {coronavirusInfo[0].information}
-        </InfoItem>
-      )}
+  render() {
+    const { category, services, isSearchCategory } = this.props;
```

Comment:
Just a nitpick, but there's something to be said for encapsulating in LocationDetails the logic of which categories start out expanded, and keeping this component focused more on what that looks like.

It makes no functional difference, just a matter of naming semantics: "isSearchCategory" means CategoryCard knows that the the searched card should start out expanded, whereas a name like "isInitiallyExpanded" or "defaultExpanded" would leave the knowledge of why that is to the component passing the prop.

(This will make more difference once we want to determine expanded cards not just by search category, but also search string etc. An abstract prop that just tells CategoryCard whether to start out expanded would mean only LocationDetails has to change, whereas "isSearchCategory" would require a change in both LocationDetails and CategoryCard.)